### PR TITLE
Dev esp32 900 td + SX127x diversity support

### PR DESCRIPTION
**Pull request by jlpoltrack**
- Adds Generic 900 True Diversity PA Target
- Updates SX127x Driver to allow for diversity
- Functionally tested by Enzaii on Discord
- Power uses values from ELRS

### CODE REVIEW

File: mLRS/Common/sx-drivers/sx127x_driver.h [hunk 1]
```diff
@@ -430,9 +430,153 @@ class Sx127xDriver : public Sx127xDriverCommon
...
+
+        SetStandby(); // should be in STDBY after reset
+        delay_us(1000); // is this needed ????
+    }
+
+    //-- high level API functions
+
+    void StartUp(tSxGlobalConfig* global_config)
+    {
+//XX        // this is not nice, figure out where to place
```
Comment (olliw42) [inline mLRS/Common/sx-drivers/sx127x_driver.h:548]: you can delete these comments here, it's sufficient to have them in teh driver above :)

**Comment by olliw42** [comment]
looks excellent, just one possible minor adjustment

when you or Tom merge, pl take care to change the commit title to something starting with `esp, ...`

(it affects a non-esp library, but I think that's so canonical I would consider this a pure esp merge :)

MAYN THX !!

File: mLRS/Common/sx-drivers/sx127x_driver.h (hunk 1 above)
Comment (jlpoltrack) [reply mLRS/Common/sx-drivers/sx127x_driver.h]: Removed comments from StartUp function

**Comment by tmcadam** [comment]
Looks great. This looks like a really interesting piece of hardware to add. 

Great to get end user test on this one too!

**Comment by jlpoltrack** [comment]
@tmcadam Could you kindly do the merge if you approve? (Try not to merge my own PRs :) )

**Comment by tmcadam** [comment]
@jlpoltrack Done!

**Pull request merged**